csharp-htk/obj/Debug/netstandard2.0/HTK.AssemblyInfoInputs.cache

```diff
@@ -0,0 +1 @@
+62b3153406622242165e51d622b7cc23ee632f9f
```

Comment:
Should all of the `.cache` files be removed? Possibly add a `.gitignore` (https://github.com/github/gitignore) file and ignore `*.cache`?